x-pack/plugins/canvas/server/routes/shareables/download.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { readFileSync } from 'fs';
+import { SHAREABLE_RUNTIME_FILE } from '../../../../../legacy/plugins/canvas/shareable_runtime/constants';
+import { RouteInitializerDeps } from '../';
+import { API_ROUTE_SHAREABLE_RUNTIME_DOWNLOAD } from '../../../../../legacy/plugins/canvas/common/lib/constants';
+
+export function initializeDownloadShareableWorkpadRoute(deps: RouteInitializerDeps) {
+  const { router } = deps;
+  router.get(
+    {
+      path: API_ROUTE_SHAREABLE_RUNTIME_DOWNLOAD,
+      validate: false,
```

Comment:
why no validation on this route?

Reply:
It's not required for this route. We don't expected anything to be passed in as part of the request.